controllers/grid/articleGalleys/form/ArticleGalleyForm.inc.php

```diff
@@ -69,38 +69,19 @@ function fetch($request) {
 		));
 		$templateMgr->assign(array(
 			'supportedLocales' => $journal->getSupportedLocaleNames(),
-			'enablePublicGalleyId' => $journal->getSetting('enablePublicGalleyId'),
 			'submissionId' => $this->_submission->getId(),
 		));
 
 		return parent::fetch($request);
 	}
 
 	/**
-	 * Validate the form
-	 */
-	function validate($request) {
-		// Check if public galley ID is already being used
-		$journal = $request->getJournal();
-		$articleGalleyDao = DAORegistry::getDAO('ArticleGalleyDAO'); /* @var $journalDao JournalDAO */
-
-		$publicGalleyId = $this->getData('publicGalleyId');
-		if ($publicGalleyId && $articleGalleyDao->pubIdExists('publisher-id', $publicGalleyId, $this->_articleGalley?$this->_articleGalley->getId():null, $journal->getId())) {
-			$this->addError('publicGalleyId', __('editor.publicIdentificationExists', array('publicIdentifier' => $publicGalleyId)));
-			$this->addErrorField('publicGalleyId');
-		}
-
-		return parent::validate();
-	}
```

Comment:
Yes, the whole code for publisher-id was moved into the identifiers tab, handled by the controllers/tab/pubIds/form/PublicIdentifiersForm.inc.php and this validation here: https://github.com/pkp/pkp-lib/blob/master/controllers/tab/pubIds/form/PKPPublicIdentifiersForm.inc.php#L159-L162